android/app/build.gradle

```diff
@@ -84,12 +84,12 @@ project.ext.react = [
 apply from: "../../node_modules/react-native/react.gradle"
 apply from: "../../node_modules/react-native-vector-icons/fonts.gradle"
 
-if (System.getenv("SENTRY_ENABLED") == "true") {
-    project.ext.sentryCli = [
+project.ext.sentryCli = [
```

Comment:
For some reason, if this block is inside the 'if' block at line 92, the 'upload sourcemap' task wouldn't run.  Moving it outside and having the environment variable 'SENTRY_ENABLD' set in the bash_profile worked for the sourcemap upload

Reply:
This was working just fine, why is this not working?

Reply:
reverted